src/websocket.c

```diff
@@ -159,8 +175,7 @@ ws_handshake_reply(struct http_client *c) {
 	p += sizeof(template4)-1;
 
 	/* send data to client */
-	ret = write(c->fd, buffer, sz);
-	(void)ret;
+	ws_write_all(c->fd, buffer, sz);
```

Comment:
The return value of this function should be propagated to the caller in `worker.c`. Can you put back the `int ret;` and use it?

Also, to pass a `server` struct to be used in `slog`, change to:
```c
ret = ws_write_all(c->s, c->fd, buffer, sz);
```

(also see comment below)